music21/musicxml/xmlToM21.py

```diff
@@ -1407,6 +1407,8 @@ def rightsToCopyright(self, rights):
 
 # -----------------------------------------------------------------------------
 class PartParser(XMLParserBase):
+    staff_type_map = {}
```

Comment:
This presumably needs to be inside the __init__ since it shouldn't be shared among PartParsers?